src/main/java/io/axoniq/axonserver/connector/admin/impl/AdminChannelImpl.java

```diff
@@ -85,6 +89,48 @@ public AdminChannelImpl(ClientIdentification clientIdentification,
         this.userServiceStub = UserAdminServiceGrpc.newStub(channel);
     }
 
+    public ResultStream<EventProcessor> eventProcessors() {
+        AbstractBufferedStream<EventProcessor, Empty> results = new AbstractBufferedStream<EventProcessor, Empty>(
+                "", BUFFER_SIZE, REFILL_BATCH
+        ) {
+
+            @Override
+            protected Empty buildFlowControlMessage(FlowControl flowControl) {
+                // no app-level flow control available on this request
+                return null;
+            }
+
+            @Override
+            protected EventProcessor terminalMessage() {
+                return EventProcessor.newBuilder().build();
+            }
+        };
+        eventProcessorServiceStub.getAllEventProcessors(EMPTY, results);
+        return results;
+    }
+
+    public ResultStream<EventProcessor> eventProcessorsByApplication(String application) {
+        Application request = Application.newBuilder().setApplication(application).build();
+
+        AbstractBufferedStream<EventProcessor, Empty> results = new AbstractBufferedStream<EventProcessor, Empty>(
+                "", BUFFER_SIZE, REFILL_BATCH
+        ) {
+
+            @Override
+            protected Empty buildFlowControlMessage(FlowControl flowControl) {
+                // no app-level flow control available on this request
+                return null;
+            }
+
+            @Override
+            protected EventProcessor terminalMessage() {
+                return EventProcessor.newBuilder().build();
+            }
+        };
+        eventProcessorServiceStub.getEventProcessorsByApplication(request, results);
```

Comment:
In other parts, we use the word "component". Suggest to rename to "getEventProcessorsByComponent".